docs/sql_reference/src/asciidoc/_chapters/sql_functions_and_expressions.adoc

```diff
@@ -3596,6 +3647,144 @@ SALES REP
 ```
 
 <<<
+[[is_ipv4_function]]
+== IS_IPV4 Function
+
+For a given argument, if it is a valid IPV4 string, IS_IPV4() returns 1 else returns 0. 
+```
+IS_IPV4( expression )
+```
+
+* `_expression_`
++
+specifies an expression that determines the values to include in the

```

Comment:
This paragraph doesn't look right to me. It looks like a discussion of an aggregate function, which IS_IPV4 is not (as far as I know). Perhaps it should say, "specifies a string expression to be checked" and nothing more?